.github/workflows/deploy.yml

```diff
@@ -35,4 +35,4 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         build_dir: website/build
-        branch: gh-pages
\ No newline at end of file
+        branch: gh-pages
```

Comment:
nani

Reply:
try running
```
git add --renormalize .
```

Reply:
I just removed the newlines in those files manually